core/src/themes/test/css-variables/index.html

```diff
@@ -983,12 +983,18 @@ <h2>Street Fighter II</h2>
 
         // The default and dark themes are official Ionic
         // themes so they are located outside of this test
-        if (theme == 'default' || theme == 'dark') {
+        let cssFile = `${themeFilesDir}/${theme}.css`;
+
+        // TODO FW-5862 update this to not
```

Comment:
Noticed that the tech debt ticket doesn't mention this, only the `colors` test 👀

Reply:
Good catch. Fixed!